### PR TITLE
fix(DB/SAI): Zul'Farrak - Adds creature_formations for trash in the last boss room

### PR DESCRIPTION
**Pull request by kjack9**
## Changes Proposed:
-  adds creature formations for trash in the last boss room of ZF

Affected GUIDs:

Sul'lithuz Abomination (Blue)
- 81527*
- 81565

Sandfury Guardian (Red)
- 37998*
- 37999
- 38000
- 38001

Bosses are Yellow and do not pull with either group.

image

## Issues Addressed:

- adds pull separately when they should all be linked together

## SOURCE:

- Sul'lithuz Abomination: https://youtu.be/I9dOA-PktnM?t=1743
- Sandfury Guardian: https://youtu.be/I9dOA-PktnM?t=1769

## Tests Performed:

- tested in-game, adds pull in correct groups without aggro'ing the boss

## How to Test the Changes:

1. `.go xyz 1767.853394 1067.639038 45.800163 209 3.907347`
2. Pull one Sul'lithuz Abomination and observe the other one aggro
3. Pull one side of the Sandfury Guardians and obverse the other side aggro

## Known Issues and TODO List:

None

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Kitzunu — CHANGES_REQUESTED** [review]
Always delete before an inert.

You can also compact it to 1 insert query

**Comment by Kitzunu** [comment]
https://www.azerothcore.org/wiki/sql-standards

**Comment by kjack9** [comment]
> Always delete before an inert.
> 
> You can also compact it to 1 insert query

You won't accept my HeidiSQL copy/paste? =P

I'll get it edited and up to standard, thanks for keeping me honest.

File: data/sql/updates/pending_db_world/rev_1684953661179621591.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+-- Sul'lithuz Abomination and Sandfury Guardian creature formations
+DELETE from `creature_formations` WHERE `memberGUID` IN (81527, 81565, 37998, 37999, 38000, 38001);
+INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`) VALUES
+(81527, 81527, 0, 0, 3, 0, 0),
+(81527, 81565, 0, 0, 3, 0, 0),
+(37998, 37998, 0, 0, 3, 0, 0),
+(37998, 37999, 0, 0, 3, 0, 0),
+(37998, 38000, 0, 0, 3, 0, 0),
+(37998, 38001, 0, 0, 3, 0, 0);
+
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1684953661179621591.sql:11]:
```suggestion
-- Sul'lithuz Abomination and Sandfury Guardian creature formations
DELETE FROM `creature_formations` WHERE `memberGUID` IN (81527, 81565, 37998, 37999, 38000, 38001);
INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`) VALUES
(81527, 81527, 0, 0, 3, 0, 0),
(81527, 81565, 0, 0, 3, 0, 0),
(37998, 37998, 0, 0, 3, 0, 0),
(37998, 37999, 0, 0, 3, 0, 0),
(37998, 38000, 0, 0, 3, 0, 0),
(37998, 38001, 0, 0, 3, 0, 0);
```

**Review by Gultask — APPROVED** [review]

**Pull request merged**